### PR TITLE
Bumped PHP requirement to 5.5

### PR DESCRIPTION
**Pull request by JeroenDeDauw**
Fixes https://github.com/wmde/WikibaseDataModelSerialization/issues/200

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
 ## 2.3.0 (2017-02-07)
 
-* Improved performance of `StatementDeserializer`.
-* Improved type safety throughout the code.
-* Reworked `DeserializerBaseTest` into `DispatchableDeserializerTest`.
-* Reworked `SerializerBaseTest` into `DispatchableSerializerTest`.
+* Improved performance of `StatementDeserializer`
+* Improved type safety throughout the code
+* Renamed `DeserializerBaseTest` to `DispatchableDeserializerTest`
+* Renamed `SerializerBaseTest` to `DispatchableSerializerTest`
```
Comment (thiemowmde) [inline RELEASE-NOTES.md]: This is not correct. It's a very different base class now, not just a rename.

File: composer.json [hunk 2]
```diff
@@ -27,12 +27,13 @@
...
 	},
 	"require": {
-		"php": ">=5.3.0",
+		"php": ">=5.5.0",
 		"wikibase/data-model": "~6.0|~5.0|~4.2",
 		"serialization/serialization": "~3.1",
 		"data-values/serialization": "~1.0"
 	},
 	"require-dev": {
+		"phpunit/phpunit": "~4.8",
```
Comment (thiemowmde) [inline composer.json]: Do you mind if I bring this in alphabetic order?

File: .travis.yml [hunk 3]
```diff
@@ -7,12 +7,10 @@ matrix:
...
-      php: 5.3
-    - env: DM=~6.0
-      php: 5.4
-    - env: DM=@dev
-      php: 5.5
+      php: 7.1
     - env: DM=~5.0
+      php: 5.5
+    - env: DM=@dev
       php: 5.6
```
Comment (thiemowmde) [inline .travis.yml]: You switched the two DataModel versions PHP 5.5 and 5.6 use. Not that it matters, just wondering. A special reason to do this?

Comment (JeroenDeDauw) [inline .travis.yml]: no

File: composer.json (hunk 2 above)
Comment (JeroenDeDauw) [inline composer.json]: no

File: RELEASE-NOTES.md (hunk 1 above)
Comment (JeroenDeDauw) [inline RELEASE-NOTES.md]: Right... "reworked" is so ambiguous tho

Perhaps this should not even be here - if it is an implementation detail the users should not worry about (no breaking change) then why have it here?

**Comment by manicki** [comment]
+1
Looks OK. I am just confused how this PHP/DM version matrix should look like so I'd better tet @thiemowmde have his eye on it.

**Pull request merged**